flutter_appauth/ios/Classes/OIDExternalUserAgentIOSNoSSO.m

```diff
@@ -0,0 +1,241 @@
+/*! @file OIDExternalUserAgentIOSNoSSO.m
+    @brief OIDExternalUserAgentIOSNoSSO is a custom user agent based off of the default user agent in the AppAuth iOS SDK found here:
+           https://github.com/openid/AppAuth-iOS/blob/master/Source/iOS/OIDExternalUserAgentIOS.h
```

Comment:
This file is actually based on the implementation file at https://github.com/openid/AppAuth-iOS/blob/master/Source/iOS/OIDExternalUserAgentIOS.m not the header file that is currently written here